### ISSUE TITLE
No entity images when Lovelace adapter runs with authentication

### ISSUE THREAD

**Issue by agross** [ioBroker/ioBroker.lovelace#88]
**Describe the bug**  
Displaying images fails as soon as Lovelace anonymous access is disabled.

**To Reproduce**  
1. Enable Lovelace instances authentication
1. Add a `glance` card
    ```yaml
    type: glance   
    entities:
      - light.Bathroom
   ```
    This forces to load e.g. `http://raspberrypi:8091/adapter/zigbee/img/ikea_bulb_E27.png` in my installation
4. No images show up

Can also be reproduced with curl:

```sh
$ curl http://raspberrypi:8091/adapter/zigbee/img/ikea_bulb_E27.png
{"error":"Not authenticated"}
```

**Expected behavior**  
Images load.

**Versions:**  
 - Adapter version: 10.15
 - JS-Controller version: 2.2.9
 - Node version: v10.20.1

**Comment by Garfonso** [comment]
Not sure how we can fix that... 😿 
It seems that lovelace does not add ?token=... parameter to picture downloads. This might also be an issue for userdefined icons (but if they use the cards-method, then the URL will not be checked for authorization by lovelace, if I read the code correctly).

So maybe we should allow unauthorized access to entity_picture values that we configure in _processCommon? Can that create security holes? @GermanBluefox ?

**Comment by agross** [comment]
Still an issue.

**Comment by Garfonso** [comment]
Thanks for the feedback. If I find time, I will look into this, again.

**Comment by Garfonso** [comment]
Hi,

could you please try with my development version? I.e. in admin adapters tap, select install from github, then custom URL and paste: `Garfonso/iobroker.lovelace#dev`
That should install a test version which should solve your issue. If so, please report back. Thanks

Garfonso

**Comment by agross** [comment]
Hi @Garfonso,

I installed your dev branch and it appears not to be working w.r.t. this issue. One question is what you store in the `_entityIconUrls` array. I added some logging to the `_readAllEntities` function and found that the `entity_picture` URLs all start with a dot, so they are file system paths:

```
2021-01-16 14:08:24.727  - info: lovelace.0 (29400) Pic: ./adapter/zigbee/img/philips_hue_color.png
```

All request URLs (`req.url`) are relative to ioBroker's web server, though so the ["does not need auth" check](https://github.com/Garfonso/ioBroker.lovelace/commit/62ec67ce7aa487ae1b1dba78e789e3769411161a#diff-1b48bb429de959672dadab9c87c215e19177dd70e3bd932d131ce74294c44cd5R1753) must fail:

```
2021-01-16 14:11:04.504  - info: lovelace.0 (29614) Req URL: /adapter/zigbee/img/ikea_bulb_E27.png
```

My suggestion: Drop the leading dot when pushing items to the `_entityIconUrls` array.

FYI: This error was logged during startup:

```
error: lovelace.0 (28330) Cannot parse themes: Error: Function yaml.safeLoad is removed in js-yaml 4. Use yaml.load instead, which is now safe by default
```

Alex
